### DISCUSSION TITLE
RE: To set up Renovate Bot to schedule jobs for each repository at different time

### DISCUSSION THREAD

**Discussion by patrickpannn** [renovatebot/renovate#25679] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Cloud (bitbucket.org)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hi,

I am just trying to work out a solution to schedule the renovate bot at different time for each repository. I notice that there is in-repository schedule by setting up the specific time for each repo. But I am not sure how this is going to work. I am currently using kubernetes to create a Cronjob, which do the job for all the repositories altogether at the scheduled time. What I want to achieve is to customise the the scheduled time for each repository. Is that doable?

### What you tried so far.

Currently still at research stage.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Not possible. Why would you want to do that? Renovate processes one repo at a time anyway

**Reply by rjclark** [reply]
I have a sort-of similar use case.

- I run Renovate from a Kubernetes cronjob nightly.
- I have about 40 self-hosted repos that Renovate is configured to look at.

For some of the repos I would like updates and PRs every night. For some of them it does not make sense and I'd like to get PRs on those weekly or monthly.

What I thought I could do (and which I have not been able to make work yet) is

1. Renovate pod uses a `config.json` with some global defaults and the list of repos to check ( _this is working currently_ )
3. A `renovate.json` in each repo that has a schedule specifically for that repo if it needs to be less frequent than daily ( _this I have not been able to get working yet_ ) 

I tried something as naive as this in my `renovate.json` in the repo I was testing, but when renovate runs it processes this repo every night. It may not be creating PRs (still need to wait on that) but the logs show that it is hitting Github and Docker Hub for all the dependencies in this repo.

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "schedule": "on the first day of the month",
}
```

***Question*** : Is the schedule just to control PR creation? I was wondering if the need to do PRs for vulnerabilities would require doing the check every time instead of on a more infrequent schedule? If that's the case I think I will end up setting up different cronjobs for the container with different repos in the config.

**Reply by rarkins** [reply]
Schedule controls branch/PR creation
